snapshot/cli/cli.go

```diff
@@ -136,6 +142,94 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 	return nil
 }
 
+// Subcommand for creating and uploading git bundles.
+// This is a workaround for creating arbitrary git bundles and keeping them in a Bundlestore.
+// We need this for now because generic bundles do not fit well with the existing
+// Snapshot, ID, Stream schemas.
+type createGitBundleCommand struct {
+	file  string
+	basis string
+	ref   string
+	ttld  time.Duration
+	keep  bool
+}
+
+func (c *createGitBundleCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create_git_bundle",
+		Short: "Creates a git bundle file as specified by basis and reference for the cwd repo",
```

Comment:
Might as well add that this uploads, and do the same for ingestGitCommitCommand.